Dockerfile

```diff
@@ -176,11 +180,22 @@ ARG CACHEID
 COPY --from=apt-dependencies /tmp/apt /
 COPY --from=dependencies /tmp/deps /usr
 COPY --from=install /tmp/cl /usr
-COPY --chown=${USER}:${USER} ./python /python
+COPY --chown=${USER}:${USER} ./python/include /python/include
+COPY --chown=${USER}:${USER} ./python/source /python/source
+COPY --chown=${USER}:${USER} ./python/pyproject.toml ./python/setup.py /python/
 RUN --mount=type=cache,target=${HOME}/.cache,id=pip-${TARGETPLATFORM}-${CACHEID},uid=1000 \
   python3 -m pip install --prefix=/tmp/python /python
 RUN mv /tmp/python/local /tmp/python-usr
 
+FROM cpp-test as python-test
```

Comment:
Seems like this could be part of `test`? Is there a specific time we might want to run just part of the tests?

Reply:
Most of time in development, we develop in cpp first and then we write the python bindings. So we want to write and test cpp first before we continue with python.

Reply:
OK, just seems like it could stay the same stage with all the tests. As long as you keep the C++ tests running first it would be a similar experience. Anyway, not a blocker.